DOC/components.md

```diff
@@ -1,9 +1,20 @@
-Case 1: Architect\
-Use case: Selecting building of interest
-* Coordinates of buildings
-  * Component specs go here!
-* User interface (textbox?) that receives specific building coordinate (position)
-* Building specs (height, general building size)
+# Case 1: Architect
```

Comment:
Regarding the component, I believe it isn't essential to delineate each USER case, as there's a possibility of overlap among users with the same use case. Overall, thank you for putting this together - it's a solid and well-organized foundation.